### PR TITLE
Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by petergoldstein**
Migrates CI to GitHub Actions, as Travis CI.org is no longer active.

Loosened/upgraded versions for bundler and rake allowed by gemspec to enable spec runs.  Everything runs green on my fork.

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (dpep) [inline .github/workflows/ci.yml:3]: is it worth scoping the [pull_request events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) more?  I usually stick to `opened`, `edited`, and `synchronize`

File: .github/workflows/ci.yml
Comment (dpep) [inline .github/workflows/ci.yml:13]: support for ruby 2.5 and 2.6 have [ended officially](https://endoflife.date/ruby)...should we remove those versions?

**Review by dpep — APPROVED** [review]
thanks for pushing to get CI up and running again!

File: .github/workflows/ci.yml
Comment (petergoldstein) [reply .github/workflows/ci.yml:3]: I don't usually bother.

File: .github/workflows/ci.yml
Comment (petergoldstein) [reply .github/workflows/ci.yml:13]: At the end of the day that's a maintainer decision.  That said, I think it would make sense.

Also, generally speaking I like to bump the gem's major version if I'm going to remove support for a Ruby version.

Comment (kennyadsl) [reply .github/workflows/ci.yml:13]: I think we can remove the unsupported versions and release a new major.

**Comment by kennyadsl** [comment]
@petergoldstein Thanks, I resolved some gemspec conflicts (you can rebase and remove my commit if you want). I'd like to merge this once we remove the unsupported Ruby versions.

**Comment by petergoldstein** [comment]
@kennyadsl I've rebased and removed the 2.5/2.6 gems.  Everything runs green on my fork.

**Comment by kennyadsl** [comment]
Thanks, I released 1.0.0

**Comment by petergoldstein** [comment]
@kennyadsl You probably want to update the README, which says Ruby 2.0+

**Comment by kennyadsl** [comment]
Yes, I updated the README but.. I forgot to update the gemspec. Sending a PR now to address that, I'll release 2.0.1 after it's merged.

**Pull request merged**
